### ISSUE TITLE
mnist benchmark

### ISSUE THREAD

**Issue by mousecpn** [mousecpn/MachineLearning_HW_CQUT#2]
for sharing the test set performance in mnist and the training experience

**Comment by choushunn** [comment]
优化器: SGD(lr=0.01, momentum=0.5)
batch_size = 64
test_acc: 98.47%

**Comment by Aflexiblebox** [comment]
1.Conv1 1->10 5*5
2.Relu
3.Maxpooling
4.Conv2 10->20 3*3
5.Linear1  2000->500
6.Relu
7.Linear2  500->10

优化器: Adam
batch_size = 64
test_acc: 94.27%

**Comment by hakunacc** [comment]
1.Conv2D(1-->6)
2.relu
3.maxpooling
4.Conv2D(6-->16)
5.relu
6.maxpooling
7.liner(320-->84)
8.liner(84-->10)

优化器: Adam
batch_size = 64
test_acc: 99.18%

**Comment by eanson023** [comment]
模型结构:
1. Conv2d(1–>32,kernel_size=(5, 5), stride=(1, 1), padding=(2, 2),activation=‘relu’)
2. Conv2d(32—>64, kernel_size=(3, 3))
3. bn(64)
4. Relu
5. maxpooling
6. DeepWise_PointWise_Conv(64–>8,activation=‘relu’)
7. linear(8*11*11–>10,dropout=0.2,activation='softmax')


优化器:SDG(lr=0.02)
batch_size = 64
test_acc: 98.33%
